pandas/core/groupby/groupby.py

```diff
@@ -199,6 +199,15 @@ class providing the base-class of operations.
 Series or DataFrame
     Computed {fname} of values within each group.
 
+See Also
+--------
+Series.min : Return the min of the group values.
+DataFrame.min : Return the min of the group values.
+Series.max : Return the max of the group values.
+DataFrame.max : Return the max of the group values.
+Series.sum : Return the sum of the group values.
+DataFrame.sum : Return the sum of the group values.
```

Comment:
Why "group values" if you're linking to e.g. `Series.min`?

Reply:
Maybe just `Series values` or `DataFrame values` as appropriate?

Reply:
Thanks for catching that, I referenced the wrong function, should have been the groupby versions of those functions.

Reply:
@rhshadrach let me know if this is appropriate or not.